packages/m4.rb

```diff
@@ -30,6 +17,9 @@ def self.build
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    Dir.chdir "#{CREW_DEST_PREFIX}/bin" do
+      FileUtils.symlink 'm4', 'gm4'
+    end
```

Comment:
What is the advantage of doing ths?

Reply:
autotools configure scripts look for `gm4` before looking for `m4`. Also, it's a little alias for users from BSD systems used to typing `gm4` when referring to GNU m4